overlay/files/system/oem/10_accounting.yaml

```diff
@@ -35,6 +35,6 @@ stages:
             Defaults !insults
             root ALL=(ALL) ALL
             %admin ALL=(ALL) NOPASSWD: ALL
-            @includedir /etc/sudoers.d
+            #includedir /etc/sudoers.d
```

Comment:
uhm, that means that sudo in the ubuntu image is old. That might also explain why tests are failing?

Reply:
See: https://github.com/kairos-io/kairos/actions/runs/3079520846/jobs/4976230587#step:4:1460